src/Microsoft.Cci.Extensions/project.json

```diff
@@ -0,0 +1,36 @@
+﻿{
+    "dependencies": {
+        "System.AppContext": "4.0.0-beta-*",
```

Comment:
Yeah I will trim this down, it has way too much unnecessary stuff in it right now.